packages/store-engine-fs/src/index.ts

```diff
@@ -0,0 +1,252 @@
+/*
+ * Wire
+ * Copyright (C) 2018 Wire Swiss GmbH
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see http://www.gnu.org/licenses/.
+ *
+ */
+
+import {CRUDEngine} from '@wireapp/store-engine';
+import {
+  PathValidationError,
+  RecordAlreadyExistsError,
+  RecordNotFoundError,
+  RecordTypeError,
+  UnsupportedError,
+} from '@wireapp/store-engine/dist/commonjs/engine/error/';
+import * as fs from 'fs-extra';
+import * as path from 'path';
+
+export class FileEngine implements CRUDEngine {
+  [index: string]: any;
+
+  public storeName = '';
+  public options: {fileExtension: string} = {
+    fileExtension: '.dat',
+  };
+  // Using a reference to Node.js' "path" module to influence the platform-specific behaviour in our tests
+  public static path: any = path;
```

Comment:
Is this still needed?

Reply:
Unfortunately yes. During the tests we need to influence if `path.posix` should be used or not. Since the import is at the beginning of the file we cannot dynamically exchange it so instead of changing the import we change the `path: any` reference.